tests/test_qnode.py

```diff
@@ -14,7 +14,7 @@
 """
 Unit tests for the :mod:`pennylane` :class:`QNode` class.
 """
-
+import pytest
 import unittest
 import logging as log
 log.getLogger('defaults')
```

Comment:
We do not probably need this now. Since pytest gives quite detailed information about the tests and we are not using any of the logging methods from before.

Reply:
Yep. This will still be needed for now, as the first half of the test is still unittest style.